src/help/help_impl.cpp

```diff
@@ -337,6 +337,16 @@ std::vector<topic> generate_topics(const bool sort_generated,const std::string &
 		} else {
 			WRN_HP << "Found a topic generator that I didn't recognize: " << generator << "\n";
 		}
+
+		if(parts.size()>1 && part[1] == "era"){
```

Comment:
I think `part[1]` is a typo for `parts[1]`.

Reply:
Fixed the typo!

Reply:
@Pentarctagon trying to contribute to resolving issue 5036

Reply:
Alright, in that case, if you put text in the commit message like "Fixes #5036", then github will automatically close that issue when that commit is merged.

Reply:
@Pentarctagon Thank you! I plan on working on it a bit more to possibly improve it.